src/TestInfrastructure/app.config

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@tvsonar I've had a look at the build output and there aren't any warnings being produced in relation to this file, so I'd just leave it as it is. It's a test project so it's not critical.